src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -84,28 +150,22 @@ public virtual void Build(string basePath, string baseName, string sourceAssembl
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.WinUI", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.UI", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.UI.Text", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.Foundation", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.UI.Composition", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.UI.Dispatching", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.UI.Input", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.UI.Windowing", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.ApplicationModel.Resources", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.Graphics", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.Phone.PhoneContract", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.Networking.Connectivity.WwanContract", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.ApplicationModel.Calls.CallsPhoneContract", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.UI.Xaml.Hosting.HostingContract", StringComparison.Ordinal)
-						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Microsoft.Web.WebView2.Core", StringComparison.Ordinal)
+						  let fileNameWithoutExtension = Path.GetFileNameWithoutExtension(externalRedfs.Display)
+						  where fileNameWithoutExtension.StartsWith("Windows.Foundation", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Microsoft.WinUI", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Microsoft.UI", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Microsoft.Foundation", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Microsoft.ApplicationModel.Resources", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Microsoft.Graphics", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Windows.Phone.PhoneContract", StringComparison.Ordinal)
+						  || fileNameWithoutExtension.StartsWith("Windows.Networking.Connectivity.WwanContract", StringComparison.Ordinal)
```

Comment:
I removed some redundant `StartsWith` checks here.

For example, we had previously `StartsWith("Microsoft.UI")`, `StartsWith("Microsoft.UI.Text")` checks. But the second is redundant. The first one is sufficient alone.

Reply:
Looks good. It was probably an incorrect duplicate.